development/docker-compose.yml

```diff
@@ -32,6 +32,15 @@ services:
     env_file:
       - ./dev.env
     tty: true
+  celery_worker:
+    image: "networktocode/nautobot-dev-py${PYTHON_VER}:local"
+    entrypoint: "nautobot-server celery worker -B -l INFO"
```

Comment:
Should we add a `health_check` entry to the `celery_worker` service, similar to what's currently there for the RQ `worker` service?

Reply:
Yes.